io.github.streetpea.chiaki4deck.yaml

```diff
@@ -0,0 +1,130 @@
+app-id: io.github.streetpea.Chiaki4deck
```

Comment:
this doesn't match the app-id found in the upstream repository.

Reply:
is that a requirement? I changed it because I do not own `re.chiaki.Chiaki4deck`.

Reply:
app-id have to match everywhere. who own `chiaki.re`. the original project? Then your fork should change that to match whatever you use.

Reply:
this also include the name of the appdata file. This is the current build failure.

Reply:
I changed the appdata file to the correct name so the build should get past its current failure.